### PR TITLE
Add autoRIFT golden test pairs for L4,5,7,9

### PR DESCRIPTION
**Pull request by jhkennedy**

### CODE REVIEW

File: hyp3_testing/templates/autorift_golden.json.j2 [hunk 1]
```diff
@@ -88,5 +88,45 @@
...
+       ]
+    },
+    "job_type": "AUTORIFT"
+  },
+  {
+    "name": "{{ name }}",
+    "job_parameters": {
+      "granules": [
+        "LT05",
+        "LT05"
```
Comment (jhkennedy) [inline hyp3_testing/templates/autorift_golden.json.j2]: @forrestfwilliams @jacquelynsmale can you add one of the Landsat 5 pairs you've been working with to add L5 support?

File: hyp3_testing/templates/autorift_golden.json.j2 [hunk 2]
```diff
@@ -88,5 +88,45 @@
...
+       ]
+    },
+    "job_type": "AUTORIFT"
+  },
+  {
+    "name": "{{ name }}",
+    "job_parameters": {
+      "granules": [
+        "LT04",
+        "LT04"
```
Comment (jhkennedy) [inline hyp3_testing/templates/autorift_golden.json.j2]: @forrestfwilliams @jacquelynsmale can you add one of the Landsat 4 pairs you've been working with to add L4 support?

File: hyp3_testing/templates/autorift_golden.json.j2 (hunk 1 above)
Comment (jacquelynsmale) [reply hyp3_testing/templates/autorift_golden.json.j2]: We were working with one scene:  LT05_L1TP_018013_20060610_20200901_02_T1 so far. I think LT05_L1TP_018013_20060626_20200831_02_T1 or LT05_L1GS_018013_20060407_20200901_02_T2 could be a good secondary or reference scene.

File: hyp3_testing/templates/autorift_golden.json.j2 (hunk 2 above)
Comment (jacquelynsmale) [reply hyp3_testing/templates/autorift_golden.json.j2]: @jhkennedy @forrestfwilliams this one was a bit trickier to select an example. We haven't been working with a specific L4 example, and searching for pairs was a bit harder. I selected a pair from Antarctica with cloud cover < 50 for both scenes. I'm not 100% confident in the quality here, but it's a pair. 
LT04_L1GS_217110_19900115_20200916_02_T2 and LT04_L1GS_217110_19891230_20200916_02_T2.

Comment (jacquelynsmale) [reply hyp3_testing/templates/autorift_golden.json.j2]: I did some more searching in the pair list and the explorer, and found a pair that has cloud cover under 5% and I verified it is in our pair list for L4. The pair name is: `LT04L1GS0501171990012901T2_LT04L1GS0501171989031501T2_03031_G0240V01_P002.nc`, which are the scenes LT04_L1GS_050117_19890315_20200916_02_T2 and LT04_L1GS_050117_19900129_20200916_02_T2.

File: hyp3_testing/templates/autorift_golden.json.j2 (hunk 1 above)
Comment (jacquelynsmale) [reply hyp3_testing/templates/autorift_golden.json.j2]: Oddly enough, this pair wasn't in the pair list provided to us by ITS_LIVE. I went back through the pair list and selected a pair with <5% cloud cover in Greenland. The pair name is `'LT05L1GS0010131992062801T2_LT05L1GS0010131992042501T2_32624_G0240V01_P008.nc'`, so the scenes are `T05_L1GS_001013_19920628_20200914_02_T2` and `LT05_L1GS_001013_19920425_20200915_02_T2`

**Comment by jhkennedy** [comment]
The landsat-4 pair here failed with an `OutOfMemoryError: Container killed due to memory usage` for all 3 attempts (the log just abruptly ends which is indicative of a memory failure; confirmed by looking at the batch job)
https://hyp3-test-api.asf.alaska.edu/jobs/5efd7f58-0cdf-4663-bed7-68d7b00b16ee

I'll dig up a L4 pair we know runs.

**Comment by jhkennedy** [comment]
@forrestfwilliams any thoughts as to why the L4 pair failed? Looks to be during the FFT step.

File: hyp3_testing/templates/autorift_golden.json.j2 [hunk 3]
```diff
@@ -88,5 +88,45 @@
...
+       ]
+    },
+    "job_type": "AUTORIFT"
+  },
+  {
+    "name": "{{ name }}",
+    "job_parameters": {
+      "granules": [
+        "LT04_L1GS_050117_19890315_20200916_02_T2",
+        "LT04_L1GS_050117_19900129_20200916_02_T2"
```
Comment (jhkennedy) [inline hyp3_testing/templates/autorift_golden.json.j2]:
```suggestion
        "LT04_L1TP_063018_19880611_20200917_02_T1",
        "LT04_L1TP_063018_19880627_20200917_02_T1"
```
This pair ran fine in the datacube test. I've run it again in production now that hyp3-autorift is released:
https://hyp3-api.asf.alaska.edu/jobs/879efe81-adf2-48cb-89db-f4bd88c577e4

**Comment by jhkennedy** [comment]
Submitted all the proposed pairs here to production:
https://hyp3-api.asf.alaska.edu/jobs?name=Golden%20test%20update%20for%20l457
```json
{
  "jobs": [
    {
      "execution_started": true,
      "job_id": "75e52782-505b-4127-9107-66218c6691af",
      "job_parameters": {
        "granules": [
          "LC09_L1GT_215109_20220125_20220125_02_T2",
          "LC09_L1GT_215109_20220210_20220210_02_T2"
        ]
      },
      "job_type": "AUTORIFT",
      "name": "Golden test update for l457",
      "priority": 9996,
      "request_time": "2022-11-03T00:36:26+00:00",
      "status_code": "PENDING",
      "user_id": "jhkennedy"
    },
    {
      "execution_started": true,
      "job_id": "87278854-a560-4bab-8004-4f79c1f5e73f",
      "job_parameters": {
        "granules": [
          "LT05_L1GS_001013_19920425_20200915_02_T2",
          "LT05_L1GS_001013_19920628_20200914_02_T2"
        ]
      },
      "job_type": "AUTORIFT",
      "name": "Golden test update for l457",
      "priority": 9998,
      "request_time": "2022-11-03T00:36:26+00:00",
      "status_code": "PENDING",
      "user_id": "jhkennedy"
    },
    {
      "execution_started": true,
      "job_id": "d0e55049-c7cd-444b-87b9-6fef0d31b276",
      "job_parameters": {
        "granules": [
          "LE07_L1TP_063018_20040911_20200915_02_T1",
          "LE07_L1TP_063018_20040810_20200915_02_T1"
        ]
      },
      "job_type": "AUTORIFT",
      "name": "Golden test update for l457",
      "priority": 9997,
      "request_time": "2022-11-03T00:36:26+00:00",
      "status_code": "PENDING",
      "user_id": "jhkennedy"
    },
    {
      "execution_started": true,
      "job_id": "879efe81-adf2-48cb-89db-f4bd88c577e4",
      "job_parameters": {
        "granules": [
          "LT04_L1TP_063018_19880627_20200917_02_T1",
          "LT04_L1TP_063018_19880627_20200917_02_T1"
        ]
      },
      "job_type": "AUTORIFT",
      "name": "Golden test update for l457",
      "priority": 9999,
      "request_time": "2022-11-03T00:36:26+00:00",
      "status_code": "PENDING",
      "user_id": "jhkennedy"
    }
  ],
  "next": "https://hyp3-api.asf.alaska.edu/jobs?name=Golden+test+update+for+l457&start_token=eyJqb2JfaWQiOiAiZWEyNzkyMzgtM2M0Yi00YWI0LThkOWUtZWQ0OWZhZGZiYzkzIiwgInVzZXJfaWQiOiAiamhrZW5uZWR5IiwgInJlcXVlc3RfdGltZSI6ICIyMDIyLTAyLTE4VDIyOjI5OjQyKzAwOjAwIn0%3D"
}
```

**Comment by jhkennedy** [comment]
Woops, accidentally summited the L4 pair with the same reference and secondary scenes. Here's the correct pair, as in the suggestion:
https://hyp3-api.asf.alaska.edu/jobs/4a74c527-f8d4-4ddf-b2ad-a3385f814153

**Review by asjohnston-asf — APPROVED** [review]

**Pull request merged**